sdk/ml/azure-ai-ml/azure/ai/ml/entities/__init__.py

```diff
@@ -360,6 +364,7 @@
     "Workspace",
     "WorkspaceKeys",
     "Connection",
```

Comment:
One more thing we need to think about. AutoML folks mentioned that the expiermental warning (from Connection) will show up when ml_client.workspaces.get is called, but ml_client.workspaces.get is not an experimental method. They suggested to remove the experimental warning.

Reply:
Just checked their notebook. I guess the experimental warning is probably not from `ml_client.workspaces.get`. Instead, it should be from `output["Subscription ID"] = ml_client.connections._subscription_id`

![image](https://github.com/Azure/azure-sdk-for-python/assets/11171669/2995eb26-253a-45e6-8cd2-b92ca3243f86)